### DISCUSSION TITLE
Kanban views

### DISCUSSION THREAD

**Discussion by almirsarajcic** [3Stones-io/omedis#261] [General]
Talking to @amos-kibet we realized the regular Kanban board view is getting bigger so it's getting hard to orient ourselves.
I've created a view for us developers to focus more easily. We're only concerned with 3 columns:
- ready (issues that have sufficient info so we can pick them up)
- in progress
- in review
It's available here: https://github.com/users/wintermeyer/projects/1/views/7.

There are also some existing views that might be useful to other people. For example, @wintermeyer could use https://github.com/users/wintermeyer/projects/1/views/5 whenever he starts working so he can see what's on his plate.
After leaving a comment/review on the PR, you can move it back to In progress so the developer can fix it and move back to In review.

Maybe we need another view just for @fradarr and @wintermeyer so they can see the big picture and make sure there's always some stuff in Ready column, so developers have something to work on.

**Comment by amos-kibet** [comment]
The Developer View is very helpful, thanks!
